packages/atomic/src/components/facets/atomic-segmented-facet-scrollable/atomic-segmented-facet-scrollable.pcss

```diff
@@ -0,0 +1,9 @@
+@import '../../../global/global.pcss';
+
+.wrapper-segmented {
```

Comment:
those can be a new mixin in `packages/atomic/src/global/polyfills.pcss` since they are polyfilling a hidden scrollbar

```
@mixin hide-scrollbar;
```